core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -611,6 +635,11 @@ case class MLFuncsStageDuration(
     stageIds: Array[Int]
 )
 
+case class MLFuncsSpeedupAndDuration(
+    averageSpeedUp: Double,
```

Comment:
nit: averageSpeedUp -> averageSpeedup (to be consistent with "speedup" treated as single word elsewhere